Sources/Kitura/Router.swift

```diff
@@ -239,14 +236,14 @@ extension Router : RouterMiddleware {
 
         let index = urlPath.index(urlPath.startIndex, offsetBy: mountpath.characters.count)
 
-        request.parsedURL.path = urlPath.substring(from: index)
```

Comment:
The following changes are API breaking changes as you aren't maintaining the old parsedURL.path contents. The idea was to deprecate the old stuff.....

This is true with the whole set of similar changes here....

The fact that parsedURL is now a computed variable doesn't help, as you made it a lazy computed variable. I don't think it gets recomputed on every access.